### ISSUE TITLE
Filter for the locals strings to translate

### ISSUE THREAD

**Issue by e-nouri** [anthonynsimon/parrot#22]
The list of strings can be pretty big in some project, the idea is that I will add a small filter in the SPA locals components to filter out and sort the list.

**Comment by anthonynsimon** [comment]
Yes, I agree. There is also a related issue #9 in which we talked about possible solutions. Maybe we can brainstorm what could be the best way to approach this.

**Comment by e-nouri** [comment]
yes definitely, how do you collect ideas or chat ?!

**Comment by anthonynsimon** [comment]
It could be here so that other contributors can see everything easily :)
